app/models/user.rb

```diff
@@ -1,6 +1,19 @@
 require 'digest/sha1'
 
 class User < ApplicationRecord
+  apipie :class, desc: "A class representing #{model_name.human} object" do
+    sections only: %w[all additional]
+    property :id, Integer, desc: 'Numerical ID of the User'
+    property :firstname, String, desc: 'Returns the user first name'
+    property :lastname, String, desc: 'Returns the user Last name'
+    property :description, String, desc: 'Returns the user description'
+    property :ssh_keys, array_of: SshKey, desc: 'Returns an array of SshKey object'
+    property :ssh_authorized_keys, array_of: SshKey, desc: 'Returns an array of all the ssh authorized_keys'
```

Comment:
This is an array of hashes representing the user's keys with their metadata (key, type and comment), not actual SshKey objects

Reply:
I think this is an array of strings: https://github.com/theforeman/foreman/blob/c3560417748c744d6f64336b1b98102e6a8bb4c1/app/models/concerns/user_usergroup_common.rb#L2-L4
https://github.com/theforeman/foreman/blob/c3560417748c744d6f64336b1b98102e6a8bb4c1/app/models/ssh_key.rb#L55-L58